src/api/app/views/layouts/webui/webui.html.haml

```diff
@@ -65,11 +65,15 @@
           = render partial: 'layouts/webui/footer', locals: { spider_bot: @spider_bot }
         .container-fluid.py-2#sponsors
           = render SponsorsComponent.new
-        .container-fluid.mt-2.py-2#footer-legal
-          %p.text-center
+        .d-flex.justify-content-center.mt-2.py-2#footer-legal
+          %div
             The Open Build Service is an
             = succeed '.' do
               = link_to('openSUSE project', 'https://www.opensuse.org')
+          - if @configuration.code_of_conduct.present?
+            .ms-2
+              = succeed '.' do
+                = link_to('Code of Conduct', code_of_conduct_index_path)
```

Comment:
I think I would rather place this in `layouts/webui/footer` in the `Help` section.

Reply:
I didn't put it in the `Help` section because I didn't consider the "Code of Conduct" help...

Reply:
It kind of is, it's documentation on what is considered "good behavior" on this instance...

Reply:
I moved the link to the `Help` section.